releasenotes/notes/use-tls-1.2-by-default-e2983c21056c3359.yaml

```
@@ -0,0 +1,5 @@
+---
+upgrade:
+    The ``force_tls_12` configuration parameter is removed in favor of ``min_tls_version``.
+    The ``min_tls_version`` configuration parameter now defaults to `tlsv1.2` instead of ``tlsv1.0``.
+    Set ``min_tls_version`` value to `tlsv1.0` or `tlsv1.1` to allow a lower TLS version.
```

I think we should put an emphasize  on the main change which is defaulting to TLS 1.2

WDYT ?
```suggestion
    The Agent now default to TLS 1.2 instead of TLS 1.0. The ``force_tls_12`` configuration parameter has been
    removed since it's now the default behavior.
    To continue using TLS 1.0 or 1.1, you must set the ``min_tls_version`` configuration parameter to either
    `tlsv1.0` or `tlsv1.1`.
```

---

Yes that's better, thanks ! I'm pinging docs to get a final review on this